### PR TITLE
drop PHP 5.4 check

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
rebased, should be merge-able now

**Comment by samsonasik** [comment]
@mwillbanks merge-able ?

File: src/Escaper.php [hunk 1]
```diff
@@ -41,13 +41,11 @@ class Escaper
 
     /**
      * Holds the value of the special flags passed as second parameter to
-     * htmlspecialchars(). We modify these for PHP 5.4 to take advantage
-     * of the new ENT_SUBSTITUTE flag for correctly dealing with invalid
-     * UTF-8 sequences.
+     * htmlspecialchars().
      *
      * @var string
```
Comment (Ocramius) [inline src/Escaper.php]: Is it actually a string?

Comment (samsonasik) [inline src/Escaper.php]: should be nullint ?

Comment (Ocramius) [inline src/Escaper.php]: Probably cannot be `null`, given the constructor structure

Comment (samsonasik) [inline src/Escaper.php]: ok, updated ;)

**Pull request merged**